doc/news/changes.h

```diff
@@ -409,10 +409,10 @@ inconvenience this causes.
   (Nicola Giuliani, 2015/04/01)
   <li>
 
-  <li> New: A new function to get a map with all vertices at boundaries
-  has been added in GridTools::get_all_vertices_at_boundary.
-  This function will return a map wich can be used in functions
-  like GridTools::Laplace_transform.
+  <li> New: A function to get a map with all vertices at boundaries has
+  been added at GridTools::get_all_vertices_at_boundary. This function will
```

Comment:
Can you add the opening and closing parentheses at the end of the function name? That way, doxygen will find it.